pkg/contract/ibchandler/ibchandler.go

```diff
@@ -225,7 +225,7 @@ type VersionData struct {
 
```

Comment:
TIPS: If it's clearly necessary, you need not execute `make abigen`.

Reply:
Revert